index.html

```diff
@@ -114,7 +114,7 @@ <h1 class="heading-primary">BarCamp 14</h1>
             </a>
             <a
               class="button share-link"
-              href="https://twitter.com/intent/tweet?url=http%3A%2F%2F2023.barcampphilly.org&text=INSERT%20TEXT%20HERE%20YIPPEE&hashtags=apple%2Cbanana"
+              href="https://twitter.com/intent/tweet?url=http%3A%2F%2F2023.barcampphilly.org&hashtags=apple%2Cbanana"
```

Comment:
What's up with the `apple banana` hashtag?

Reply:
Ah that was a placeholder one too, forgot about that 😄. will remove...!

Reply:
602e0f7..d8c56c2